README.md

```diff
@@ -180,6 +180,11 @@ export default {
 }
 ```
 
+> If you are using babel 6 compiler and up you need to do the following in order for setup to work
```

Comment:
- `babel 6 compiler` => `Babel >= v6` 
- in order for setup  => in order for *the* setup to work